![[Pasted image 20210812213729.png]]
Recombination DNA with 20-40bp homology by 5' __T5 exonuclease activity__ in one isothermal reaction with participation of polymerase and ligase. 

The homolgous ends have to be added (f.i. by [[PCR]]) to both fragments that you want to fuse. 5' --> 3' exonuclease activity creates 3' overhangs. These complementary sequences then anneal, creating the dsDNA of interest. __DNA polymerase__ fills the gaps, __Taq ligase__ seals the nicks. 

In contrary to [[Recombineering#Lambda Red Recombineering|Lambda Red Recombineering]] this is a in vitro method. 

In contrary to [[Sequence- and ligase-independent Cloning (SLIC)|SLIC]], Gibson assembly uses a dedicated exonuclease which is heat liable.

Application: [Enzymatic assembly of DNA molecules up to several hundred kilobases](https://www.nature.com/articles/nmeth.1318)


## Advantages
- Allows for simple assemble of multiple fragments in the chosen orientation (up to 6 DNA fragments in one reaction)
- Doesn't create any unwanted sequences at the junctions
- Any DNA fragment can be used if it has the appropriate overhangs
- Single-tube one-step reaction (SLIC isn't necessarily)
- Multiple mutations can be introduced by primer design
- Long stretches of synthetic DNA can be made --> Was used for the creation of [[Mycoplasma mycoides]](https://www.wikiwand.com/en/Mycoplasma_mycoides)
- Combinatorial gene synthesis: Variants of fragments (red and black arrows) can be used in the same macroreaction to create enormous amounts of fragment combinations, resulting in different gene variants, varying in function and activity
![[Pasted image 20210812215237.png]] 

## Disadvantages
- Only works with segments longer than 200bp, as shorter ones have the potential to be chewed back completely by the 5' exonuclease.