- Sequencing by synthesis with reversible terminators
- Uses fluorescently labeled nucleotides which are removable
- 300bp read lengths
- Base Confidence: ca. 95%

## Process
![[Pasted image 20210815100424.png]]
- Libraries are denatured and bound at one end to a solid surface coated with __adapter oligonucleotides__. The free end of each fragment bends over and hybridizes to a complementary adapter on the surface which initiates complementary strand synthesis (PCR). 
- Multiple cycles of this solid-phase amplification followed by denaturation create clusters of 1000 copies of ssDNA molecules
- The sequencing step uses reversible fluorescent nucleotides. Upon incorporation of a nucleotide its specific fluorophore flashes up. The 3' terminator on the base and the fluorophore are removed
- For a given cluster all identical strands are read simultaneously
- This happens asynchronously
![[Pasted image 20210815102403.png]]



## Reagents
- Primers
- DNA pol
- Four differently labeled [[reversible terminator nucleotides]]

## Advantages
• leader among NGS platforms
• highest throughput
• lowest per-base cost
• wide range of applications
## Disadvantages
• requires PCR amplification for library preparation
• short read length (300 bp)
• technically challenging: overloading -> over-lapping clusters and poor sequence quality.
• requirement for sequence complexity.