- Individual DNA polymerase molecules immobilized at the bottom of 100nm nano-wells
- The fluorophore of the labeled nucleotides is excited upon formation of the new phospho-diester bond, whose emission is picked up by the __Zero-Mode Waveguide (ZMW)__ optics. 4 colors


![[Pasted image 20210815123748.png]]

![[Pasted image 20210815124523.png]]
A hexa(!)-phosphate group (red arrow). As a result of this attachment, the fluorophore is released when the nucleotide is incorporated into the growing DNA strand while a transient fluorescent signal is produced at the enzyme’s active site. Also note the long linker and large central triazin-based scaffold (bottomleft), which helps place the fluorescence moiety (topleft) away from the polymerase surface. This reduces the chance for quenching and photochemical reactions between the fluorophore and the enzyme, which increases the strength of the fluorescent signal and reduces the risk of chemical damage of the polymerase.
![[Pasted image 20210815124600.png]]
Fluorescence time trace from a single nano well of a PacBiosequencer. The individual peaks correspond to the fluorescent signal from a single nucleotide being integrated into the growing DNA strand. Note the large variation in the duration from peak to peak. The section shown here corresponds to ca. 4sec. A full trace for a 30’000 nucleotide read has a duration of ca. 2.5h.

## Advantages
- Long reads (<20kbp), limited by the detection volume of 20x10^-15 ul --> De novo genome assemblies
- Fast run times (few hours)
- Detection of epigenetic modifications
## Disadvantages
- High costs
- High error rates at single pass (ca. 14%)
- Low throughput, because low parallelization
- Limited range of applications