   # RNA-Seq
RNA sequencing refers to techniques used to determine the sequence of RNA-derived cDNA molecules using [[NGS]]. RNA is isolated from a biological sample, cDNA is generated by reverse transcription, followed by Next-Gen Sequencing to determine sequence and relative abundance of all cDNA molecules. It therefore is a sequencing method to __quantitatively__ analyse the __transcriptome__ (all active genes) for a given physiological condition or developmental stage.

## (Dis-)advantages/Applications
 - It quantifies alternative splicing (in eukaryotes). 
 - The transcripts don't have to be predescribed like in the other methods. 
 - It allows for the discovery of new transcripts. 
 - Additional insights from RNA-Seq datasets can be gained which can't be gained from microarrays. 
 - It provides information on the structure of operons, for instance what genes are clustered together on an operon. 
 - It also permits the discovery of non-coding RNAs such as Antisense RNAs which play a role in gene regulation. 
 - It allows for the identification of alternative transcriptional start sites within operons, overlapping untranslated regions and [riboswitches](https://en.wikipedia.org/wiki/Riboswitch).
 - [[Translatome identification]] 

![[Pasted image 20210812120022.png]]

## Process
![[Pasted image 20210812115326.png]]
- Ribosomal RNA (rRNA) has to be removed prior to cDNA library preparation, as 80% of the total RNA consists of rRNA. The __Ribozero Kit__ is commonly used for that. 
- After that the mRNA is processed for sequencing: The mRNAs are fragmented and RNA or DNA adaptors ligation adapters are ligated to both ends using special __T4 RNA ligases__, which are able to direct their enzymatic activity either to the 5' or 3' end of the mRNA. These adapters contain optimal priming sites for the subsequent __reverse transcription__ reaction. "T4 RNA ligase 2 truncated" specifically ligates the pre-adenylated 5' ligation adapter to the 3' end of the mRNA. Unlike the full length enzyme, this truncated version can't ligate to the 5' end of the mRNA. It doesn't require any ATP, but can only use pre-adenylated adapters and thus creates background ligation. No concatomers or circles are formed. The first ligation adapter bears a blocked 3' end (no hydroxy group at the end) which prohibits any further ligations. In the second ligation step a 5' ligation adapter is ligated to the 5' end of the mRNA by T4 RNA ligase 1. The 5' end of this adapter is also blocked as it lacks a phosphate group. 
- __Reverse transcription__ reaction can then perceed using a primer that is complementary to the 3' adapter. 
- This creates ssDNA which is taken under [[PCR]] amplification using oligonucleotides which (importantly) have the __sequencing adapters (Illumina fw and rv adapters)__ attached to them, ligated to both ends of the ssDNA to create dsDNA. 
- This ds cDNA is then ready to be loaded on the NGS platform. Here the sequenced DNAs are aligned to the reference genome and counted. The number of transcripts can then be mapped along the locus on the genome. Note that the fw strands (+) are mirrored to the rv strands (-).
![[Pasted image 20210812115805.png]]
